packages/roles/roles_common_async.js

```diff
@@ -1052,8 +1052,13 @@ Object.assign(Roles, {
    * @return {Promise<Cursor>} Cursor of users in roles.
    */
   getUsersInRoleAsync: async function (roles, options, queryOptions) {
+    options = Roles._normalizeOptions(options)
+    
+    const assignmentOptions = { ...options }
+    delete assignmentOptions.queryOptions
```

Comment:
```suggestion
   assignmentOptions.queryOptions = undefined 
```

I think we try not to use `delete` (cc @italojs)

Reply:
delete statement isnt a good practice for "hot code" and roles would be high requested in big projects that uses meteor.

`Delete` statement prevent V8 JIT to optmize this funciton since it change the original schema in runtime 

References: 
- https://leapcell.io/blog/unlocking-node-js-performance-through-smarter-v8-jit-interactions
- https://www.thenodebook.com/node-arch/v8-engine-intro#myth-1-v8-is-just-an-interpreter

[Benchmark](https://github.com/ishtms/v8-perf)

Reply:
@italojs interesting read, thanks for pointing that out, wasn't aware of that; will review my own codebase now as well... ;)